reference/7/Microsoft.PowerShell.Utility/Out-GridView.md

```diff
@@ -0,0 +1,427 @@
+---
+external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Utility
+ms.date: 10/28/2019
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/out-gridview?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: Out-GridView
+---
+# Out-GridView
+
+## SYNOPSIS
+Sends output to an interactive table in a separate window.
+
+## SYNTAX
+
+### PassThru (Default)
+
+```
+Out-GridView [-InputObject <PSObject>] [-Title <String>] [-PassThru] [<CommonParameters>]
+```
+
+### Wait
+
+```
+Out-GridView [-InputObject <PSObject>] [-Title <String>] [-Wait] [<CommonParameters>]
+```
+
+### OutputMode
+
+```
+Out-GridView [-InputObject <PSObject>] [-Title <String>] [-OutputMode <OutputModeOption>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Out-GridView` cmdlet sends the output from a command to a grid view window where the output is
+displayed in an interactive table.
+
+Because this cmdlet requires a user interface, it does not work on Server Core installations of
+Windows Server.
+
+You can use the following features of the table to examine your data:
+
+- Hide, Show, and Reorder Columns: To hide, show, or reorder a column, right-click a column header
+  and then click Select Columns.
```

Comment:
Guidance is to avoid "click" and use something like "Select" instead as not all uses will use a mouse in a graphical interface. Is there a shortcut or key combo to do this, also?